src/media/css/base.styl

```diff
@@ -154,5 +154,42 @@ body {
 }
 
 #site-footer {
```

Comment:
Then we're just using a class as an ID. We aren't going to run into any specificity issues on those particular elements, so I see no argument for not just using an ID.